src/common/ScrollableContainer/test/ScrollableContainer.spec.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import {
+  createRendererWithUniDriver,
+  cleanup,
+} from '../../../../test/utils/unit';
+import { scrollableContainerDriverFactory } from '../ScrollableContainer.uni.driver';
+import ScrollableContainer from '../ScrollableContainer';
+import { getScrollPositionY } from '../scrollPositionLogic';
+
+describe('ScrollableContainer', () => {
+  const render = createRendererWithUniDriver(scrollableContainerDriverFactory);
+
+  afterEach(() => {
+    cleanup();
+  });
```

Comment:
Remove an unnecessary closure:
```suggestion
  afterEach(cleanup);
```